### PR TITLE
Add docstring examples

### PR DESCRIPTION
**Pull request by bschifferer**
Added docstring examples https://github.com/NVIDIA/NVTabular/issues/273

### CODE REVIEW

**Comment by bschifferer** [comment]
I added docstring examples similar to DifferenceLag/CosineSimilarity.

@rnyak proposed to provide real small examples, e.g.

`
df = cudf.DataFrame({'A': [1, 2, 2, 6, 6, 8], 'B': [2, 2, 4, 7, 7, 2], 'C': [1,4, 4, 8, 8, 5]})
right = cudf.DataFrame({'A': [1, 2, 7], 'D': [2, 7, 5]})
`

should I create dummy dataframes in the example and apply the op to?

File: nvtabular/ops/lambdaop.py [hunk 1]
```diff
@@ -24,6 +24,49 @@ class LambdaOp(TransformOperator):
...
+        )
+
+    Example usage 2:
+
+        # Initialize the workflow
+        proc = nvt.Workflow(
+            cat_names=CATEGORICAL_COLUMNS,
+            cont_names=CONTINUOUS_COLUMNS,
+            label_name=LABEL_COLUMNS
+        )
```
Comment (benfred) [inline nvtabular/ops/lambdaop.py:55]: I don't think we need to define the workflow twice here
```suggestion
    Example usage 2::
```

File: nvtabular/ops/lambdaop.py [hunk 2]
```diff
@@ -24,6 +24,49 @@ class LambdaOp(TransformOperator):
...
+
+        # Add LamdaOp to the workflow and
```
Comment (benfred) [inline nvtabular/ops/lambdaop.py:57]: typo 
```suggestion
        # Add LambdaOp to the workflow and
```

File: nvtabular/ops/logop.py [hunk 3]
```diff
@@ -27,6 +27,18 @@ class LogOp(TransformOperator):
     to handle the common case of zerofilling null values, this is
     calculates ``log(1+x)`` instead of just ``log(x)``.
 
+    Example usage:
```
Comment (benfred) [inline nvtabular/ops/logop.py:30]:
```suggestion
    Example usage::
```

File: nvtabular/ops/normalize.py [hunk 4]
```diff
@@ -31,6 +31,18 @@ class Normalize(DFOperator):
 
     It performs Normalization using the mean std method.
 
+    Example usage:
```
Comment (benfred) [inline nvtabular/ops/normalize.py:34]:
```suggestion
    Example usage::
```

File: nvtabular/ops/normalize.py [hunk 5]
```diff
@@ -76,6 +88,18 @@ class NormalizeMinMax(DFOperator):
 
     It performs Normalization using the min max method.
 
+    Example usage:
```
Comment (benfred) [inline nvtabular/ops/normalize.py:91]:
```suggestion
    Example usage::
```

File: nvtabular/ops/join_groupby.py [hunk 6]
```diff
@@ -33,6 +33,23 @@ class JoinGroupby(DFOperator):
...
+            cont_names=CONTINUOUS_COLUMNS,
+            label_name=LABEL_COLUMNS
+        )
+
+        # Add JoinGroupby to the workflow
+        proc.add_feature(
+            JoinGroupby(
+                columns=['cat1', 'cat2', 'cat3'], # columns which are groupby
+                cont_names=['num1'], # continous column, which the statistics are applied to
+                stats=['sum','count']), # statistcs, which are applied
```
Comment (benfred) [inline nvtabular/ops/join_groupby.py:50]: typo
```suggestion
                stats=['sum','count']), # statistics, which are applied
```

File: nvtabular/ops/fill.py [hunk 7]
```diff
@@ -61,6 +74,18 @@ class FillMedian(DFOperator):
...
+    Example usage:
+
+        # Initialize the workflow
+        proc = nvt.Workflow(
+            cat_names=CATEGORICAL_COLUMNS,
+            cont_names=CONTINUOUS_COLUMNS,
+            label_name=LABEL_COLUMNS
+        )
+
+        # Add FillMedian to the workflow for continous columns
```
Comment (benfred) [inline nvtabular/ops/fill.py:86]: typo
```suggestion
        # Add FillMedian to the workflow for continuous columns
```

File: nvtabular/ops/fill.py [hunk 8]
```diff
@@ -25,6 +25,19 @@ class FillMissing(DFOperator):
...
+    Example usage::
+
+        # Initialize the workflow
+        proc = nvt.Workflow(
+            cat_names=CATEGORICAL_COLUMNS,
+            cont_names=CONTINUOUS_COLUMNS,
+            label_name=LABEL_COLUMNS
+        )
+
+        # Add FillMissing to the workflow for continous columns and specify fill value
```
Comment (benfred) [inline nvtabular/ops/fill.py:37]: typo
```suggestion
        # Add FillMissing to the workflow for continuous columns and specify the fill value
```

File: nvtabular/ops/join_groupby.py [hunk 9]
```diff
@@ -33,6 +33,23 @@ class JoinGroupby(DFOperator):
...
+            cat_names=CATEGORICAL_COLUMNS,
+            cont_names=CONTINUOUS_COLUMNS,
+            label_name=LABEL_COLUMNS
+        )
+
+        # Add JoinGroupby to the workflow
+        proc.add_feature(
+            JoinGroupby(
+                columns=['cat1', 'cat2', 'cat3'], # columns which are groupby
+                cont_names=['num1'], # continous column, which the statistics are applied to
```
Comment (benfred) [inline nvtabular/ops/join_groupby.py:49]:
```suggestion
                cont_names=['num1'], # continuous column, which the statistics are applied to
```

File: nvtabular/ops/normalize.py [hunk 10]
```diff
@@ -31,6 +31,18 @@ class Normalize(DFOperator):
...
+
+        # Initialize the workflow
+        proc = nvt.Workflow(
+            cat_names=CATEGORICAL_COLUMNS,
+            cont_names=CONTINUOUS_COLUMNS,
+            label_name=LABEL_COLUMNS
+        )
+
+        # Add Normalize to the workflow for continous columns
```
Comment (benfred) [inline nvtabular/ops/normalize.py:43]:
```suggestion
        # Add Normalize to the workflow for continuous columns
```

File: nvtabular/ops/normalize.py [hunk 11]
```diff
@@ -76,6 +88,18 @@ class NormalizeMinMax(DFOperator):
...
+    Example usage:
+
+        # Initialize the workflow
+        proc = nvt.Workflow(
+            cat_names=CATEGORICAL_COLUMNS,
+            cont_names=CONTINUOUS_COLUMNS,
+            label_name=LABEL_COLUMNS
+        )
+
+        # Add NormalizeMinMax to the workflow for continous columns
```
Comment (benfred) [inline nvtabular/ops/normalize.py:100]:
```suggestion
        # Add NormalizeMinMax to the workflow for continuous columns
```

File: nvtabular/ops/logop.py [hunk 12]
```diff
@@ -27,6 +27,18 @@ class LogOp(TransformOperator):
...
+
+        # Initialize the workflow
+        proc = nvt.Workflow(
+            cat_names=CATEGORICAL_COLUMNS,
+            cont_names=CONTINUOUS_COLUMNS,
+            label_name=LABEL_COLUMNS
+        )
+
+        # Add LogOp to the workflow for continous columns
```
Comment (benfred) [inline nvtabular/ops/logop.py:39]:
```suggestion
        # Add LogOp to the workflow for continuous columns
```

**Review by benfred — COMMENTED** [review]
Thanks for this! Looks good aside from some minor formatting + typos

**Review by benfred — APPROVED** [review]

**Pull request merged**